source/cloud/terraform.md

```diff
@@ -347,6 +346,38 @@ EOT
 :::{warning}
 Exposing ports to the internet is potentially dangerous. Make sure that the application you're exporting is properly secured.
 :::
+(terraform_share)=
+## NFS Share
+The latest templates include a module to easily configure an NFS share.
```

Comment:
I would include an example in the templates about this and add here  ``cat xxx >> main.tf`` if they want to use it